### ISSUE TITLE
[IMPROVE] Consistent approach for flags with multiple inputs

### ISSUE THREAD

**Issue by jmaslek** [OpenBB-finance/OpenBBTerminal#2887]
When using a function that can have multiple inputs, we should specify a single way of handling.  For example.

In stocks/ca, we add using csv deliniation
```
add AAPL,MSFT
```

But in economy, we do
```
fred dgs2 dgs5
```

We need to pick one and stick to it.

**Comment by deeleeramone** [comment]
![Screen Shot 2022-10-17 at 12 14 50 PM](https://user-images.githubusercontent.com/85772166/196262938-09d0c063-bb50-4ad0-946f-02724f3dcd16.png)
![Screen Shot 2022-10-17 at 12 15 19 PM](https://user-images.githubusercontent.com/85772166/196263050-2c472125-3d0e-4a7f-9018-1439b865952d.png)

**Comment by jmaslek** [comment]
> ![Screen Shot 2022-10-17 at 12 14 50 PM](https://user-images.githubusercontent.com/85772166/196262938-09d0c063-bb50-4ad0-946f-02724f3dcd16.png) ![Screen Shot 2022-10-17 at 12 15 19 PM](https://user-images.githubusercontent.com/85772166/196263050-2c472125-3d0e-4a7f-9018-1439b865952d.png)

Since each api has a different naming convention, this is fun.  Everything should be case insensitive, so I could do UnItED-StaTes or united_states.

My thought is that we should have convention of

```
cmd -p val_one,val_two,val_three
```

When these are parameters.  So for ycrv, change to `united_states` (and process before model) and for macro it would be `united_states,united_arab_emerates,canada`

@DidierRLopes @JerBouma @montezdesousa ?

**Comment by JerBouma** [comment]
Agreed, let's go with the `comma` one. This was me being a rebel going for my own style :)

**Comment by DidierRLopes** [comment]
Agreed, let's go with it 🙏🏽
